src/custom/components/AccountDetails/index.tsx

```diff
@@ -128,6 +129,34 @@ export function getStatusIcon(connector?: AbstractConnector, walletInfo?: Connec
   return null
 }
 
+type ActivitiesGroupedByDate = {
+  date: Date
+  activities: ActivityDescriptors[]
+}[]
+
+function groupActivitiesByDay(activities: ActivityDescriptors[]): ActivitiesGroupedByDate {
+  const mapByTimestamp: { [timestamp: number]: ActivityDescriptors[] } = {}
+
+  activities.forEach((activity) => {
+    const { date } = activity
+    // get timestamp of the day, drop hours, minutes and seconds
+    const timestamp = new Date(date.getFullYear(), date.getMonth(), date.getDate()).getTime()
```

Comment:
don't we have some date library in this app? also I'm not crazy about the date format. Can we make it prettier somehow? big ask i knnow but it looks weird imo

Reply:
For what it's worth, what I applied in the base PR (now merged into presign/base from @anxo) is to just get all the timestamps in a `Date` object and then apply formatting options. See:
https://github.com/gnosis/cowswap/pull/1417/files#diff-f591f447e126d570bf5ffdc1f646a7826958496985659162101b05fe999b1f53R156

and

https://github.com/gnosis/cowswap/pull/1417/files#diff-f591f447e126d570bf5ffdc1f646a7826958496985659162101b05fe999b1f53R169

So as you mention @alfetopito a basic `util` function could suffice here (just have to normalize the input time format I suppose (detect ISO or string dates, etc.).

Reference:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/DateTimeFormat